src/cuga/backend/cuga_graph/nodes/cuga_lite/agent_graph_adapter.py

```
@@ -0,0 +1,1103 @@
+                    if not agent_id:
+                        try:
+                            from cuga.backend.server.main import app as _app
+
+                            _as = getattr(_app.state, "app_state", None)
+                            agent_id = getattr(_as, "agent_id", None) if _as else None
+                            if knowledge_config_hash is None:
+                                knowledge_config_hash = (
+                                    getattr(_as, "knowledge_config_hash", None) if _as else None
+                                )
+                        except Exception:
+                            pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)